openpype/modules/deadline/plugins/publish/collect_default_deadline_server.py

```diff
@@ -41,9 +41,12 @@ def process(self, context):
                             ["deadline"]
                             ["deadline_servers"])
         if deadline_servers:
-            deadline_server_name = deadline_servers[0]
-            deadline_webservice = deadline_module.deadline_urls.get(
-                deadline_server_name)
+            # This is wrong but either the Ayon settings schema is wrong or
+            # this logic needs to change
+            deadline_webservice = deadline_servers[0]
+            # deadline_server_name = deadline_servers[0]
+            # deadline_webservice = deadline_module.deadline_urls.get(
+            #     deadline_server_name)
```

Comment:
```suggestion
	    # treat it as a list in case of AYON otherwise use the service
	    # from the enum defined in system settings
	    # TODO: change the logic once Deadline server enum
	    # in implemented in AYON
            deadline_webservice = deadline_servers[0]
            if not AYON_SERVER_ENABLED:
            	deadline_webservice = deadline_module.deadline_urls.get(
            	    deadline_webservice)
```

Reply:
note that this also requires import `from openpype import AYON_SERVER_ENABLED`